### ISSUE TITLE
Scroll not working

### ISSUE THREAD

**Issue by FrantisekVrab** [krsyoung/redoc-rails#5]
Hi,

I installed this gem on Rails 4 with Turbolinks 5. Everything works except the scroll after the click on the menu, or menu highlighting menu items when scrolling.

How to proceed? Thanks.

**Comment by RomanHotsiy** [comment]
@FrantisekVrab,

this is ReDoc issue so open it here: https://github.com/Rebilly/ReDoc/issues

Could you also include a minimal reproducible sample? Thanks

**Comment by krsyoung** [comment]
Hey guys, I'm way behind on release versions as well, I'll update to the latest tonight.

@FrantisekVrab any chance you took a look at issue #4 to see if yarn might be a better approach (i.e. to use the NPM package directly).

@RomanGotsiy Long time no chat!

**Comment by krsyoung** [comment]
@FrantisekVrab wanted to check if you had any luck?

**Comment by RomanHotsiy** [comment]
@krsyoung 
> Long time no chat!

Definitely! Let's connect on twitter ([@RomanHotsiy](https://twitter.com/RomanHotsiy)) or wherever else
